docs/projects/ad5766_sdz/index.rst

```diff
@@ -0,0 +1,270 @@
+an external 2.5 V reference. Depending on the span selected, the mid-point of
+the output span can be adjusted allowing for a minimum output voltage as low as
+−20 V or a maximum output voltage of up to +14 V.
+The :adi:`AD5766`/ :adi:`AD5767` have integrated output buffers which can sink
+or source up to 20 mA. This makes the AD5766/AD5767 suitable for Indium Phosphide
+Mach Zehnder Modulator (InP-MZM) biasing applications.
+The part incorporates a power-on reset circuit that ensures that the DAC outputs
+power up to 0V and remain at this level until the output range of the DAC is
+configured. The outputs of all DACs are updated through register configuration,
+with the added functionality of user-selectable DAC channels to be simultaneously
+updated.
+The :adi:`AD5766`/ :adi:`AD5767` require four power supplies. AVCC is the analog
+supply for the low voltage DAC circuitry. AVDD and AVSS are the positive and
+negative high voltage power supplies for the output amplifiers. A VLOGIC supply
+pin is provided to set the logic levels for the digital interface pins.
+The :adi:`AD5766`/ :adi:`AD5767` utilize a versatile 4-wire serial interface
+that operates at clock rates of up to 50 MHz for write mode and up to 10MHz for
+readback and daisy-chain mode, and is compatible with SPIR, QSPI., MICROWIRE. 
+and DSP interface standards.
+The :adi:`AD5766`/ :adi:`AD5767` are available in a 4mm x 4mm WLCSP package
+and operates at the range of -40C to +105C.
+
+Applications:
+  * Mach Zehnder Modulator Bias Control
+  * Analog Output Modules
+  * Process Control
+
+Supported boards
+-------------------------------------------------------------------------------
+
+-  :adi:`EVAL-AD5766 <EVAL-AD5766>`
+
+Supported devices
+-------------------------------------------------------------------------------
+
+-  :adi:`AD5766`
+-  :adi:`AD5767`
+
+Supported carriers
+-------------------------------------------------------------------------------
+
+-  :xilinx:`ZedBoard <products/boards-and-kits/1-8dyf-11.html>` on FMC slot
+-  :adi:`SDP-I-FMC <EVAL-SDP-I-FMC>`
+
+Block design
+-------------------------------------------------------------------------------
+
+Block diagram
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+AD5766_SDZ
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+.. image:: ad5766_spi_engine_hdl.svg
+   :width: 800
+   :align: center
+   :alt: AD5766_SDZ block diagram
+
+Jumper setup
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+================== ================= ====================================
+Jumper/Solder link Default Position  Description
+================== ================= ==================================== 
+LK1,LK2,LK3        A                 Supplied by the 3.3 V supply (J12)
+LK4,LK5            B                 Supplied by the :adi:`ADP5071` power solution
```

Comment:
Format the table to fit the text. You can also extend the Description width (add more === until you reach the characters limit).
Another option would be using the other table format that you used at L102-120.

Reply:
Done.